migrations/847-remove-nominationmessage-column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `addons` DROP COLUMN `nominationmessage`;
```

Comment:
Note: if all tests pass on both, merge #1097 first so that we don't need to rename the migration (#1097 uses migration no 846)